testdata/src/static_msg/static_msg.go

```diff
@@ -23,13 +23,21 @@ func tests() {
 	slog.Log(ctx, slog.LevelInfo, constMsg)
 	slog.With("key", "value").Info(constMsg)
 
+	slog.Info(varMsg)                      // want `message should be a string literal or a constant`
+	slog.InfoContext(ctx, varMsg)          // want `message should be a string literal or a constant`
+	slog.Log(ctx, slog.LevelInfo, varMsg)  // want `message should be a string literal or a constant`
+	slog.With("key", "value").Info(varMsg) // want `message should be a string literal or a constant`
+
 	slog.Info(fmt.Sprintf("msg"))                      // want `message should be a string literal or a constant`
 	slog.InfoContext(ctx, fmt.Sprintf("msg"))          // want `message should be a string literal or a constant`
 	slog.Log(ctx, slog.LevelInfo, fmt.Sprintf("msg"))  // want `message should be a string literal or a constant`
 	slog.With("key", "value").Info(fmt.Sprintf("msg")) // want `message should be a string literal or a constant`
 
-	slog.Info(varMsg)                      // want `message should be a string literal or a constant`
-	slog.InfoContext(ctx, varMsg)          // want `message should be a string literal or a constant`
-	slog.Log(ctx, slog.LevelInfo, varMsg)  // want `message should be a string literal or a constant`
-	slog.With("key", "value").Info(varMsg) // want `message should be a string literal or a constant`
+	// binary expressions:
+	slog.Info("msg" + "msg")
+	slog.Info("msg" + "msg" + "msg")
+	slog.Info("msg" + constMsg)
+	slog.Info("msg" + varMsg)                                 // want `message should be a string literal or a constant`
```

Comment:
I'm unsure about this one. A variable value can change

Imagine a code like this

```go
var status = "OK"
slog.Info("status:" + status)

// something happen and now
var status = "Broken"
slog.Info("status:" + status)
```

But maybe I didn't understand what the linter reports

Reply:
That's why we report this case, note the comment on the right.

Reply:
Oh that's the message, it makes sense to me now.